cmd/snap/cmd_aliases_test.go

```diff
@@ -79,11 +80,12 @@ func (s *SnapSuite) TestAliases(c *C) {
 	c.Assert(err, IsNil)
 	c.Assert(rest, DeepEquals, []string{})
 	expectedStdout := "" +
-		"App        Alias       Notes\n" +
-		"bar.dump   bar_dump    enabled\n" +
-		"bar.???    bar_dump.1  undefined,disabled\n" +
-		"foo        foo0        auto\n" +
-		"foo.reset  foo_reset   -\n"
+		"App               Alias        Notes\n" +
+		"bar.dump          bar_dump     manual\n" +
+		"bar.dump          bar_dump.1   unaliased\n" +
+		"bar.safe-restore  bar_restore  manual,auto:restore\n" +
+		"foo               foo0         auto\n" +
+		"foo.reset         foo_reset    manual\n"
```

Comment:
these are ok, I thought you proposed "unaliased" at some point because we still the plan to have "snap install --unaliased" for example

s/auto:restore/override/ loses a bit of information but that's OK I suppose?